src/v/kafka/server/handlers/list_offsets.cc

```diff
@@ -82,45 +82,53 @@ static ss::future<list_offset_partition_response> list_offsets_partition(
        ntp = std::move(ntp),
        isolation_lvl = model::isolation_level(
          octx.request.data.isolation_level)](cluster::partition_manager& mgr) {
-          auto partition = mgr.get(ntp);
+          auto partition = mgr.get(ntp.source_ntp());
           if (!partition) {
               return ss::make_ready_future<list_offset_partition_response>(
                 list_offsets_response::make_partition(
-                  ntp.tp.partition, error_code::unknown_topic_or_partition));
+                  ntp.input_ntp().tp.partition,
```

Comment:
why we use here `input()` while we try to get `source_ntp` ?

Reply:
It actually makes no difference here, the only difference between a materialized ntp and a regular one is the topic name. I selected `input_ntp()` because that is the non-modified version of the topic, coming from the client request